java-checks/src/main/java/org/sonar/java/symexecengine/ExecutionState.java

```diff
@@ -106,10 +119,12 @@ public ExecutionState restoreParent() {
     return parent.merge(this);
   }
 
+  @Deprecated
   Set<State> getStatesOfCurrentExecutionState() {
```

Comment:
![New](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/new.png) ![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![New](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/new.png) ![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)